### DISCUSSION TITLE
Option to append /* eslint-disable */ in generated files.

### DISCUSSION THREAD

**Discussion by jimmymkude** [timostamm/protobuf-ts#255] [Ideas]
Hi all,

I find the '// tslint:disable' on top of the generated files very useful and convenient. I was wondering if the same could be for eslint. Appending '/* eslint-disable */' beneath the '// tslint:disable'.

**Comment by timostamm** [comment]
Very good point, this should definitely be done for eslint too. In fact I thought we were doing this already :grimacing:

Made it [an issue](https://github.com/timostamm/protobuf-ts/issues/256).

**Reply by jimmymkude** [reply]
Thank you! :) 🙏
